sirepo/server.py

```diff
@@ -584,7 +593,14 @@ def api_updateFolder(self):
             simulation_db.save_simulation_json(r, fixup=False)
         return self.reply_ok()
 
-    @sirepo.api.Spec("require_user")
+    @sirepo.api.Spec(
+        "require_user",
+        file="FileArg input",
```

Comment:
LibFile (no need for input)

Reply:
9d450384bbf20294b6007b678c3dde0143022f40